### DISCUSSION TITLE
NVDA reading the wrong amount of rows

### DISCUSSION THREAD

**Discussion by LittleWhiteLoti** [indiana-university/rivet-source#660] [Questions]
https://rivet.iu.edu/components/preview/default-table/#

Current NVDA Output: "Table with 5 rows and 3 columns"

Expected NVDA Output: "Table with 4 rows and 3 columns"

https://codepen.io/aardrian/pen/YEKmxP

Current NVDA Output: "Table with 7 rows and 5 columns" which is correct.

There seems to be something wrong with Rivet's table component

**Comment by scottanthonymurray** [comment]
@LittleWhiteLoti Thanks for writing, and apologies for the delayed follow-up -- our team is just getting back into the office after the holiday break. We'll pass your notes along to the accessibility team.
